pydalboard/modules/filter.py

```diff
@@ -1,23 +1,30 @@
-from typing import ClassVar
 from dataclasses import dataclass
 
 import numpy as np
 
 from pydalboard.signal import SignalInfo
 from pydalboard.modules.base import Module
 
+from enum import Enum
+
+
+class FilterType(Enum):
+    LOW_PASS = 1
+    HIGH_PASS = 2
+    BAND_PASS = 3
+
+
 @dataclass
 class FilterParameters:
     cutoff: float  # Cutoff frequency in Hz
     resonance: float  # Resonance (Q) # TODO: define possible values (+6db is ~ √2)
-    filter_type: str # Type of filter: 'low', 'high' or 'band'
+    filter_type: FilterType  # Type of filter
     slope: int  # Slope in dB/octave: 12 or 24
 
     def __post_init__(self):
-        self.b0, self.b1, self.b2, self.a1, self.a2 = self.calculate_biquad_coefficients()
-        # Initialize filters memories
-        self.prev_input = [[0.0, 0.0], [0.0, 0.0]]  # [x1[n-1], x1[n-2]], [x2[n-1], x2[n-2]]
-        self.prev_output = [[0.0, 0.0], [0.0, 0.0]]  # [y1[n-1], y1[n-2]], [y2[n-1], y2[n-2]]
+        self.b0, self.b1, self.b2, self.a1, self.a2 = (
```

Comment:
Again, can you explain to me the use of () here? I'm not sure I understand I wanna be a better Pythoner ;)

Reply:
Same as above, to split into two lines.

We can avoid this by having  `calculate_biquad_coefficients` directly set the fields' values, instead of returning them. I am okay with it either way.